packages/ckeditor5-image/src/image/imageblock.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * @license Copyright (c) 2003-2020, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module image/image/imageblock
+ */
+
+import Plugin from '@ckeditor/ckeditor5-core/src/plugin';
+
+import { viewFigureToModel } from './converters';
+
+import { toImageWidget, createImageViewElement, getImageTypeMatcher } from './utils';
+
+/**
+ * The image engine plugin.
+ *
+ * It registers:
+ *
+ * * `<image>` as a block element in the document schema, and allows `alt`, `src` and `srcset` attributes.
+ * * converters for editing and data pipelines.
+ * * `'imageInsert'` command.
+ *
+ * @extends module:core/plugin~Plugin
+ */
+export default class ImageBlock extends Plugin {
```

Comment:
This plugin should require `Widget` and `ImageTextAlternative`. And also maybe `ImageEditing` in case some integrator would use only `ImageBlock` without `Image` itself?